includes/utilities.inc

```diff
@@ -22,6 +22,34 @@
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
  */
 
+/**
+ * Autocomplete callback; return all objects that can be an association.
+ */
+function islandora_ead_get_potential_associations($search_value) {
+  $tuque = islandora_get_tuque_connection();
+  $query = <<<EOQ
```

Comment:
Any concern with the size of the RI and the time it'll take to perform this query as opposed to using Solr?